### PR TITLE
Added methods for list and delete files

### PR DESCRIPTION
**Pull request by MannarAmuthan**
Added List and Delete API samples

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -94,6 +96,24 @@ def upload_file():
...
+    print("Total number of files {} ".format(file_count))
+    files = result['files']
+    print(files)
+
+
+def delete_file():
+    storage = Storage(client)
+    print_green("Running Delete File API")
+    result = storage.list_files()
+    first_file_id = result['files'][0]['$id']
```
Comment (Nibba2018) [inline app.py:113]: The console output for `delete_file` is kinda empty so it would be better if you could print out `result`.

**Review by Nibba2018 — APPROVED** [review]
Hey @MannarAmuthan , thank you for creating this PR. I ran your changes locally and everything seems to work as expected. I have a small suggestion regarding `delete_file` method which you can find below.

File: app.py (hunk 1 above)
Comment (Nibba2018) [reply app.py:113]: Also don't forget to update `README.md`

Comment (MannarAmuthan) [reply app.py:113]: Sure will do it

Comment (MannarAmuthan) [reply app.py:113]: Done, Updated READme and added console output for delete

**Review by Nibba2018 — APPROVED** [review]

**Comment by eldadfux** [comment]
THANK YOU! All changes merged 🥳

Please reach out to me on our [Discord server](https://appwrite.io/discord) if you would like me to send you some cool Appwrite stickers and magnets! Please note that shipping might take longer times because of COVID19.

As a way of saying thank you, we have just invited you to join the Appwrite organization on GitHub. You can accept the invite by visiting https://github.com/orgs/appwrite/invitation. By joining our team, you will officially be an Appwrite maintainer on GitHub.

You can change your membership [visibility settings](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/publicizing-or-hiding-organization-membership), so your new Appwrite team membership badge will show up on your personal GitHub profile.

**Pull request merged**